index.html

```diff
@@ -1288,6 +1286,55 @@ <h2>DID Subject</h2>
 property. The value of <code>id</code> in the resolved <a>DID document</a> is
 expected to match the <a>DID</a> that was resolved.
       </p>
+        </section>
+
+        <section>
+        <h2>sameSubject</h2>
```

Comment:
I think "alsoKnownAs" was a contender for the name, here, right? I think that would be better. I prefer the triple:

```
<did:example:123> alsoKnownAs <did:example:456>
```

Over:

```
<did:example:123> sameSubject <did:example:456>
```

Which reads strangely when you think about the fact that "sameSubject" is an attribute or property of the DID subject. So, as an example, if the DID subject is Drummond, it's like saying that Drummond has an attribute "sameSubject" of "github/tallTree". Drummond doesn't have a "same subject" of "github/tallTree". Rather, "github/tallTree" is *another identifier* for Drummond.

So, I'd rather say that Drummond is also known as "github/tallTree". Using "sameSubject" seems like we're bringing in contextual information that Drummond happens to have a DID Document of which he is the subject when we're just trying to talk about his attributes. It feels more like a metadata property to me: the subject of this DID Document is the same one as this other DID Document -- like we're conflating metadata about the DID Document with properties/attributes of the DID subject itself. I think we can avoid all that with a property name like "alsoKnownAs".